CONDUCT.md

```diff
@@ -1,13 +1,44 @@
-# Contributor Code of Conduct
+---
```

Comment:
I agree. If my memory serves me correctly, they were put in to stop the Jekyll compiler complaining about lines that were too long. I can remove them though, especially where it is just me deleting whitespace.